### PR TITLE
TiSpark 201 add batch write insert tests

### LINKED ISSUE #828: add tests for batchwrite insert
add tests for batchwrite insert

### PR DESCRIPTION
**Pull request by windtalker**
fix #828 add batch write insert tests

### CODE REVIEW

File: core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala [hunk 1]
```diff
@@ -23,90 +19,179 @@ class InsertSuite extends BaseDataSourceTest("test_datasource_insert") {
     )
   )
 
-  test("Test upsert to table without primary key") {
+  def compareRow(r1 : Row, r2 : Row) : Boolean = {
```
Comment (marsishandsome) [inline core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala]: private?

File: core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala [hunk 2]
```diff
@@ -23,90 +19,179 @@ class InsertSuite extends BaseDataSourceTest("test_datasource_insert") {
...
+    r1.getAs[Int](0) < r2.getAs[Int](0)
+  }
+
+  def generateData(start : Int, length : Int, skipFirstCol : Boolean = false): List[Row] = {
```
Comment (marsishandsome) [inline core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala]: private?

File: core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala (hunk 1 above)
Comment (windtalker) [reply core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala]: done

File: core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala (hunk 2 above)
Comment (windtalker) [reply core/src/test/scala/com/pingcap/tispark/datasource/InsertSuite.scala]: done

**Pull request merged**